.github/workflows/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Run Rust tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: icepuma/rust-action@master
+        with:
+          args: cargo fmt -- --check && cargo test
```

Comment:
Could you use different steps per tool, like in here:
https://github.com/chris-zen/kiro-synth/blob/master/.github/workflows/build.yaml#L58